Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV CYPRESS_INSTALL_BINARY=0
 WORKDIR /app
 COPY frontend/ .
 RUN if [ "$BUILDARCH" != "amd64" ]; then apt-get update && apt-get install -y build-essential python3 --no-install-recommends; fi
-RUN npm install -g pnpm@9 && pnpm install --no-optional --frozen-lockfile
+RUN npm install -g pnpm@9 && pnpm install --frozen-lockfile
```

Comment:
in docker don't you need to keep it? or set the environment variable too?

Reply:
It's already there in line 6 of the dockerfile so no need to set it